doxygen/parallelism_support/opencl_support.md

```diff
@@ -180,45 +222,7 @@ Eigen::Matrix<double, -1, -1> C = from_matrix_cl(C_cl);
 In addition to the functions listed in the reverse mode list, 
 the following functions can be used with `matrix_cl<T>` arguments:
 
-- acos, acosh
-- add
 - append_col, append_row
-- asin, asinh
-- atan, atanh
-- block
-- cbrt
-- ceil
-- col
-- cols
-- cos, cosh
-- diagonal
-- digamma
-- dims
-- divide
-- elt_divide
-- elt_multiply
-- erf, erfc
-- exp, exp2, expm1
-- fabs
-- floor
-- inv, inv_logit, inv_sqrt, inv_square
-- lbeta
-- lgamma
-- log, log10, log1m_exp, log1m_inv_logit
-- log1p, log1p_exp, log2, logit
-- minus
-- operator-, operator+, operator*
-- plus
-- pow
-- rep_matrix, rep_row_vector, rep_vector
-- round
-- row
-- rows
-- sin, sinh
-- size
-- sqrt
-- tan, tanh
-- tcrossprod
-- tgamma
-- transpose
-- trunc
+- gp_exp_quad_cov
+- trace
+- trigamma
```

Comment:
Why are these separate?

Reply:
These only work with `matrix_cl<T>` and not with `var_value<matrix_cl<double>>`. So rev is not done for them yet.